arangod/Cluster/Maintenance.h

```diff
@@ -164,7 +165,8 @@ arangodb::Result diffPlanLocal(
     std::vector<std::shared_ptr<ActionDescription>>& actions,
     MaintenanceFeature::ShardActionMap const& shardActionMap,
     ReplicatedLogStatusMapByDatabase const& localLogs,
-    ReplicatedStateStatusMapByDatabase const& localStates);
+    ReplicatedStateStatusMapByDatabase const& localStates,
+    replication::Version defaultReplicationVersion = replication::Version::ONE);
```

Comment:
Is this fallback value intentional?

Reply:
we should remove it and make the version always explicit.